### PR TITLE
fpb-phobic

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
departmental FPBs are much more limited now to their department
streward is also limited to combat FPBs.
Preimer has no limit on FPBs

Heads of Staff also can only be their department fpb
Scavs have a mix of FPBs they can use depending on the job
Propis can be BS FPB and normal type
Scavs can be church/SI/Guild and normal FPB types

### CODE REVIEW

**Comment by nikothedude** [comment]
btw as i said in devchat if this is merged i think there should be some ic headsup and reasoning for it

**Comment by Trilbyspaceclone** [comment]
> TLDR:
> 
> This PR restricts GENERIC FBPs currently, which have no other advantages other than being just a bobot, with fixed stats, with no special implants from several roles rather nonsensically. (which actually makes a few players be FORCED to change their characters to match this change, which is a MASSIVE nono)
> 
> Until at least that is corrected and other jobs are reconsidered I really say this has absolutely no reason to be merged.
> 
> We do NOT have a current issue with people abusing synths in other departments than whats intended, maybe past Blackshield synths in prospies.
> 
> As Moon said.
> 
> "We are no longer balancing around powergaming."
> 
> So why are you ignoring that?



k

**Comment by Guidesu** [comment]
Mister Trolby.
I give my humble suggestion than instead of making things very restrictive.... i would say.... maybe perhaps maybe.

Let Church Synths be in ANY department (makes sense in general) (Yup that means as well garderners)
Corpsman BS or SI
And such go on.
I would try making all of the synths fit on their own little niche and letting the most space for adaptability as possibly without hindering people with their characters from sudden changes

**Comment by Guidesu** [comment]
That means uhm.
Maybe give more space for extra jobs instead of only thoses

**Comment by cdb-is-not-good** [comment]
My thoughts on this PR so far;
department synths limited to department = 👍 
Generic synths, unbranded, etc limited out of departments = 👎 
Going to leave this up until at least monday for discussion given the rather major changes it'll have for a good few people.

File: code/game/jobs/job/church.dm [hunk 1]
```diff
@@ -19,6 +19,8 @@
 	access = list(
 		access_nt_preacher, access_nt_disciple, access_morgue, access_chapel_office, access_crematorium, access_maint_tunnels, access_RC_announce, access_keycard_auth, access_heads, access_sec_doors
 	)
+	disallow_species = list(FORM_FBP, FORM_UNBRANDED, FORM_SOTSYNTH, FORM_AGSYNTH, FORM_BSSYNTH, FORM_NASHEF)
```
[the remark this answers is not in the document]
Comment (r4d6) [reply code/game/jobs/job/church.dm:22]: You know that generic FBPs doesn't support cruciforms, which you need to have in order to play as a churchie, right?

File: code/game/jobs/job/church.dm [hunk 2]
```diff
@@ -86,6 +88,7 @@
 	STAT_VIG = 10,
 	STAT_TGH = 5,
 	)
+	disallow_species = list(FORM_FBP, FORM_UNBRANDED, FORM_SOTSYNTH, FORM_AGSYNTH, FORM_BSSYNTH, FORM_NASHEF)
```
[the remark this answers is not in the document]
Comment (r4d6) [reply code/game/jobs/job/church.dm:91]: They can't join the church anyway since they can't get cruciforms.

**Comment by Trilbyspaceclone** [comment]
> Generic synths, unbranded, etc limited out of departments

It dosnt do that tho, its only for heads of staff (and artist/church)

**Comment by nikothedude** [comment]
image
my thoughts

**Comment by cdb-is-not-good** [comment]
nix the part that restricts department heads from being unbranded/'FBP' style chassis imo

**Pull request merged**